package.json

```diff
@@ -3,6 +3,7 @@
   "private": true,
   "dependencies": {
     "@rails/webpacker": "3.5",
+    "resolve-url-loader": "^3.0.0",
```

Comment:
Can you explain what `resolve-url-loader` does?

Reply:
Making Sass work with a feature based project structure.
Rewrites relative paths in url() statements based on the original source file.
For projects with a feature based structure this will be a problem, since you will want to co-locate your assets with your .scss partials
https://www.npmjs.com/package/resolve-url-loader